File: CS/WindowsApplication3/Main.cs
using DevExpress.XtraEditors;
using System;
using System.Windows.Forms;


namespace DXSample
{
    public partial class Main : XtraForm
    {
        DragDropProvider provider;

        public Main()
        {
            InitializeComponent();
        }

        private void OnFormClosing(object sender, FormClosingEventArgs e)
        {
            if(provider != null)
                provider.DisableDragDrop();
        }

        private void OnLoad(object sender, EventArgs e)
        {
            provider = new DragDropProvider(pictureEdit1);
            provider.EnableDragDrop();
        }

        private void OnSimpleButtonMouseDown(object sender, MouseEventArgs e)
        {
            SimpleButton btn = sender as SimpleButton;
            pictureEdit1.DoDragDrop(btn.Image, DragDropEffects.Copy);
        }
    }
}

File: VB/WindowsApplication3/Main.vb
Imports DevExpress.XtraEditors
Imports System
Imports System.Windows.Forms

Namespace DXSample
    Partial Public Class Main
        Inherits XtraForm
        Private provider As DragDropProvider

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub OnMainFormClosing(ByVal sender As Object, ByVal e As FormClosingEventArgs) Handles MyBase.FormClosing
            If provider IsNot Nothing Then
                provider.DisableDragDrop()
            End If
        End Sub

        Private Sub OnMainLoad(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            provider = New DragDropProvider(pictureEdit1)
            provider.EnableDragDrop()
        End Sub

        Private Sub OnSimpleButtonMouseDown(ByVal sender As Object, ByVal e As MouseEventArgs) Handles simpleButton2.MouseDown, simpleButton1.MouseDown
            Dim btn As SimpleButton = TryCast(sender, SimpleButton)
            pictureEdit1.DoDragDrop(btn.Image, DragDropEffects.Copy)
        End Sub
    End Class
End Namespace
